app/controllers/admin/base_items_controller.rb

```diff
@@ -40,7 +42,9 @@ def show
 
   def destroy
     @base_item = BaseItem.includes(:items).find(params[:id])
-    if @base_item.items.any? && @base_item.destroy
+    if (@base_item.id = KitCreateService.find_or_create_kit_base_item!.id)
+      redirect_to admin_base_items_path, alert: "You cannot delete the Kits base item. This is reserved for all Kits."
+    elsif @base_item.items.any? && @base_item.destroy
```

Comment:
Shouldn't this be `@base_item.items.empty?`

Reply:
yes, fixed in commit [94030f7](https://github.com/rubyforgood/human-essentials/pull/4665/commits/94030f75aefe8cfa6ecc4e6155da4c8e18bb630f)